src/dc_receive_imf.rs

```diff
@@ -32,17 +32,28 @@ enum CreateEvent {
     IncomingMsg,
 }
 
+pub async fn dc_receive_imf(
+    context: &Context,
+    imf_raw: &[u8],
+    server_folder: impl AsRef<str>,
+    server_uid: u32,
+    seen: bool,
+) -> Result<()> {
+    dc_receive_imf_inner(context, imf_raw, server_folder, server_uid, seen, false).await
+}
+
 /// Receive a message and add it to the database.
 ///
 /// Returns an error on recoverable errors, e.g. database errors. In this case,
 /// message parsing should be retried later. If message itself is wrong, logs
 /// the error and returns success.
-pub async fn dc_receive_imf(
+pub async fn dc_receive_imf_inner(
```

Comment:
Does this have to be public? This function probably should be `pub(crate)` and comment moved to `dc_receive_imf`, which could also be `pub(crate)`.